app/controllers/likes_controller.rb

```diff
@@ -1,31 +1,21 @@
 class LikesController < ApplicationController
   class LikesController < ApplicationController
-    before_action :authenticate_user! # Add authentication if needed
+    before_action :authenticate_user!
 
-    # Create a new like for a resource (e.g., a post)
     def create
-      @like = Like.new(like_params)
-      @like.user = current_user # Assign the current user as the one who liked
+      @like = @post.likes.build(user: @current_user)
 
       if @like.save
-        redirect_to @like.liked_item, notice: 'Like was successfully created.'
+        redirect_to @post, notice: 'Liked the post!'
       else
-        render :new
+        redirect_to @post, alert: 'Unable to like the post.'
       end
     end
 
```

Comment:
- [ ] Great work on creating the create method :+1: but it would be nice if you make the `new` method to handle your comment form :thinking: Kindly make the new method to follow the project requirements :+1: